drivers/flash/flash_stm32_ospi.c

```diff
@@ -1458,7 +1561,7 @@ static void spi_nor_process_bfp_addrbytes(const struct device *dev,
 {
 	struct flash_stm32_ospi_data *data = dev->data;
 
-	if (jesd216_bfp_addrbytes == JESD216_SFDP_BFP_DW1_ADDRBYTES_VAL_4B) {
+	if (jesd216_bfp_addrbytes == JESD216_SFDP_BFP_DW1_ADDRBYTES_VAL_3B4B) {
```

Comment:
Aren't we missing the `JESD216_SFDP_BFP_DW1_ADDRBYTES_VAL_4B`?
As 4 bytes are valid for `JESD216_SFDP_BFP_DW1_ADDRBYTES_VAL_3B4B` and for `JESD216_SFDP_BFP_DW1_ADDRBYTES_VAL_4B`.

Reply:
yes, JESD216_SFDP_BFP_DW1_ADDRBYTES_VAL_4B ORJESD216_SFDP_BFP_DW1_ADDRBYTES_VAL_3B4B